### ISSUE TITLE
Unable to create Expr instances with params

### ISSUE THREAD

**Issue by kamoo1** [lelit/pglast#97]
```python3
pglast.ast.SubLink(
                {
                    "@": "SubLink",
                    "subLinkType": pglast.enums.SubLinkType.ANY_SUBLINK,
                    "testexpr": pglast.ast.ColumnRef(
                        {
                            "@": "ColumnRef",
                            "fields": (
                                pglast.ast.String({"@": "String", "val": "tab"}),
                                pglast.ast.String({"@": "String", "val": "_id"}),
                            ),
                        }
                    ),
                    "subselect": where_in_sel,
                }
)
```

Raises
```python3
File "../venv/lib/python3.8/site-packages/pglast/ast.py", line 3394, in __init__
    super().__init__(subLinkType)
TypeError: __init__() takes 1 positional argument but 2 were given
```

There is a second definition of `Expr` with `__init__` taking no parameters, removing this seems to fix the problem.
https://github.com/lelit/pglast/blob/8b13001685794d5996b2a5a1cb8f51d5d1e05597/pglast/ast.py#L2108

**Comment by lelit** [comment]
Thank you, good catch! I will address this in the following days.

**Comment by lelit** [comment]
This is fixed in the upcoming v3.8.
